anvil/src/udev.rs

```diff
@@ -1441,19 +1441,16 @@ impl AnvilState<UdevData> {
                 }
             })
             .unwrap_or_else(|| {
-                let texture = TextureBuffer::from_memory(
-                    &mut renderer,
+                let buffer = MemoryRenderBuffer::from_slice(
                     &frame.pixels_rgba,
-                    Fourcc::Abgr8888,
+                    Fourcc::Argb8888,
```

Comment:
Are you certain this had to be swapped? `pixels_rgba` reads like we already had the correct order.